packages/livekit-server-sdk/src/EgressClient.ts

```diff
@@ -1,28 +1,31 @@
 // SPDX-FileCopyrightText: 2024 LiveKit, Inc.
 //
 // SPDX-License-Identifier: Apache-2.0
-import {
+import type {
```

Comment:
@lukasIO I'm not sure if this is a good idea to change.. but my editor did it automatically.. should I revert?

Reply:
this is fine, there's no functional change, prettier enforces this for clarity

Reply:
ok, give it a ✅ if it looks fine. we'll get it released